blueocean-dashboard/src/main/js/components/RunDetails.jsx

```diff
@@ -34,32 +38,30 @@ class RunDetails extends Component {
             config.pipeline = pipeline;
 
             this.props.fetchRunsIfNeeded(config);
-            this.props.setPipeline(config);
```

Comment:
This was another extra call to `setPipeline()` that could be problematic in firing off dup calls.